### PR TITLE
Improve README

### PR DESCRIPTION
**Pull request by vctt94**
This PR updates the README to use go 1.11 and adds the params to execute the hardforkdemo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,24 +7,27 @@ information about the tesnet hardfork voting.
...
 
+
+1. Install Go version 1.11 or higher, and Git.
+Make sure each of these are in the PATH.
+
+2. Clone this repository.
+
+3. Install hardforkdemo
 ``` bash
-go get -v github.com/golang/dep/cmd/dep
```
Comment (dajohi) [inline README.md:11]: the `go install` is not enough.  there are template files required too.

**Review by dajohi — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (vctt94) [reply README.md:11]: If installing outside GOPATH, `go install` is enough, I believe, isn't it?

Should I add the description if building inside GOPATH, as well?

Comment (dajohi) [reply README.md:11]: Did you try `$ go install github.com/decred/hardforkdemo`?

Comment (vctt94) [reply README.md:11]: Yes, I did. If running inside `hardforkdemo` directory, it works.

Does it work for you? What answer does it give?

Comment (dajohi) [reply README.md:11]: but how does one get that hardforkdemo directory now that the instructions where removed?

Comment (vctt94) [reply README.md:11]: fair point. Is it better now?

File: README.md [hunk 2]
```diff
@@ -7,24 +7,48 @@ information about the tesnet hardfork voting.
...
+
+    ``` bash
+    cd $GOPATH/src/github.com/decred/hardforkdemo
+    export GO111MODULE=on
+    go install
+    ```
+
+    3.2. If building outside of GOPATH, modules are automatically enabled, and go install is sufficient.
+    ``` bash
+    go install
```
Comment (dajohi) [inline README.md:39]: need `cd hardforkdemo`

**Review by dajohi — CHANGES_REQUESTED** [review]
must be updated to the new repo name.

**Comment by vctt94** [comment]
I believe most things have been addressed so I am closing this PR now

**Pull request closed without merging**